test/BlockChainHelper.cpp

```diff
@@ -497,7 +497,7 @@ bool TestBlockChain::addBlock(TestBlock const& _block)
 		block.sync(*m_blockChain.get());
 
 		//BOOST_REQUIRE(m_lastBlock.blockHeader().hash() == BlockHeader(block.blockData()).hash());
-		m_lastBlock.setState(block.state());
+		m_lastBlock.setState(block.fromPending((uint)m_lastBlock.blockHeader().number()));
```

Comment:
The argument to `fromPending` is not the block number but the transaction number. Are there pending transactions in that block? The modifications in the hard fork are not done by transactions.